### ISSUE TITLE
Parameterset Could not find requested hyper-parameters warning

### ISSUE THREAD

**Issue by noabartal** [allegroai/clearml#710]
Hi,
I am trying to use the ParameterSet for hyper-parameter tuning with dependencies,
an example of how i use it:
ParameterSet([{“prm1”:1, “prm2": 1},{“prm1”:2, “prm2":2}])
But i get a warning :
clearml.automation.optimization - WARNING - Could not find requested hyper-parameters [None] on base task ***
It is because the paramererSet name is None

Thanks!

**Comment by jkhenning** [comment]
Hi @noabartal , can you provide a more complete code example please?

**Comment by erezalg** [comment]
Hi @noabartal,

Is this issue still happening? If not, can we close this issue?

**Comment by noabartal** [comment]
I don't know if it is still happening, i couldn't solve the problem so i didn't use this parameter.

**Comment by erezalg** [comment]
@noabartal,

Makes sense :) If you still would like our help to solve this, please provide a minimal code example that reproduces the issue and we'll do our best to support you :)

**Comment by noabartal** [comment]
I talked to someone from your team on slack when i had this issue ( and provided the code example) and they couldn't help me. 
If i will have this issue again i will definitely use your help. Thanks!

**Comment by BalogneseCoder** [comment]
I suppose it's a lack of consistency: base class Parameter has attribute name https://github.com/allegroai/clearml/blob/0594a5ac8504d090bb851265f4f5c02c8d862dde/clearml/automation/parameters.py#L42
and ParameterSet inherited from it has hardcoded name None
https://github.com/allegroai/clearml/blob/0594a5ac8504d090bb851265f4f5c02c8d862dde/clearml/automation/parameters.py#L308
better to remove name attribute from the base class Parameter and add it where it's needed.
that's not a big problem, but sometimes confusing with warnings in console
